packages/design-system-react/package.json

```diff
@@ -35,10 +35,13 @@
     "dist/"
   ],
   "scripts": {
-    "build": "ts-bridge --project tsconfig.build.json --verbose --clean --no-references",
+    "build": "yarn generate:icons && ts-bridge --project tsconfig.build.json --verbose --clean --no-references",
```

Comment:
Adding `generate:icons` to the build script to ensure SVG icons are automatically converted to React components before the TypeScript build starts. This guarantees that all icon components are available during the build process and prevents missing component errors.